src/instance/minecraft/commands/SlayerCommand.ts

```diff
@@ -0,0 +1,113 @@
+import * as assert from 'node:assert'
+import { Slayer } from 'hypixel-api-reborn'
+import { ChatCommandContext, ChatCommandHandler } from '../common/ChatInterface'
+
+const Slayers: Record<string, string[]> = {
+  zombie: ['revenant', 'rev', 'zombie'],
+  spider: ['tarantula', 'tara', 'spider'],
+  wolf: ['sven', 'wolf'],
+  enderman: ['voidgloom', 'eman', 'enderman'],
+  blaze: ['inferno', 'demonlord', 'blaze'],
+  vampire: ['riftstalker', 'bloodfiend', 'vamp', 'vampire'],
+  overview: ['all', 'summary']
+}
+
+export default {
+  name: 'Secrets',
```

Comment:
You probably means `Slayers`
```suggestion
  name: 'Slayers',
```